### ISSUE TITLE
Request list issues since Clockwork 5.3 update

### ISSUE THREAD

**Issue by FeBe95** [underground-works/clockwork-app#120]
Hey there, thanks for updating the package & app. I noticed some weird behavior with the dev tools extension though:
- Sometimes I can't click an item in the list, only double-clicking works
- After a short amount of time, the list scrolls back up again, and selects the first item in the list

There were no errors in the debugging console for Clockwork (`about:debugging`), only some warnings about a date format being incorrect. But I don't think that this is the cause of this issue.
```
Use `y` instead of `Y` (in `Y-MM-dd HH:mm:ss`) for formatting years to the input `1732013236956.2`; see: https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md
```

It does not happen every time, and I can't really find a way to reproduce this.

## Version Info
- Windows 11
- Firefox 132.0.2 (64-Bit)
- Clockwork App 5.3
- Clockwork PHP Package 5.2
- Laravel Framework 10.48.23

## Screen Recording

Note: Every time I "hover" over an item in the recording, I am actually clicking it. But nothing happens.

https://github.com/user-attachments/assets/05c08f29-f5ab-4957-897d-691b415cafa6

**Comment by itsgoingd** [comment]
Hey, thanks for the bug report. This is now fixed in master ([740e6ea](https://github.com/underground-works/clockwork-app/commit/740e6eab3577506ebdbb576e50b5ece17954d80f)) and will be released soon.

**Comment by itsgoingd** [comment]
Fixed in Clockwork 5.3.1 (might take a few hours to get through review).
